front-end/src/screens/ProposalPost/query.ts

```diff
@@ -42,6 +42,7 @@ const proposalPost = gql`
         }
         content
         created_at
+        has_poll
```

Comment:
```suggestion
```

Reply:
typescript complaining if it is not there